module/API/Controller.h

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file Controller.h
+ * @brief モーター制御に用いる関数をまとめたラッパークラス
+ * @author aridome222
+ */
+#ifndef CONTROLLER_H
+#define CONTROLLER_H
+
+#include "ev3api.h"
+#include "Motor.h"
+#include "Clock.h"
+
+class Controller {
+ public:
+  Controller();
+  Controller(const Controller&) = delete;  // 明示的にインスタンス化を禁止
+
+  /**
+   * タイヤのモータにPWM値をセット
+   * @param pwm PWM値
+   */
+  void setRightMotorPwm(const int pwm);
+  void setLeftMotorPwm(const int pwm);
```

Comment:
コンストラクタ禁止したからstaticにしないと外部から呼び出せないかな？
```suggestion
  static void setRightMotorPwm(const int pwm);
  static void setLeftMotorPwm(const int pwm);
```